test/metabase/api/search_test.clj

```diff
@@ -600,14 +608,19 @@
         (is (= (default-results-with-collection)
                (search-request-data :crowberto :q "test"))))))
 
+  ;; TODO need to isolate these two tests properly, they're sharing  temp index
```

Comment:
I think this is another piece of debt we can leave for the future owners. Nothing major.